### ISSUE TITLE
Crash - NPE - ticking entity

### ISSUE THREAD

**Issue by sanados** [zmaster587/AdvancedRocketry#867]
Reproducable:

Warp station to Gallifrey.
Launch satellite from station.
Game crashes when sattelite "leaves" satellite monitoring.

(Attached crashreport is from the startup after the crash)

Version: 
forge-12.18.3.2477 
Age of Engineering 1.1.1c
AdvancedRocketry-1.10.2-1.0.19
LibVulpes-1.10.2-0.1.19

Also tried with current versions (replace AoE versions with):
AdvancedRocketry-1.10.2-1.2.2.jar
LibVulpes-1.10.2-0.2.5-universal.jar
Then the game starts but gas charge pad is not working anymore.
Satellites are missing or show "too far"

[crash-2017-10-07_07.46.12-server.txt](https://github.com/zmaster587/AdvancedRocketry/files/1364340/crash-2017-10-07_07.46.12-server.txt)
[crash-2017-10-07_07.49.16-server.txt](https://github.com/zmaster587/AdvancedRocketry/files/1364338/crash-2017-10-07_07.49.16-server.txt)

**Comment by zmaster587** [comment]
Does this issue continue to occur in the 1.2.x series?

**Comment by sanados** [comment]
was trying to say that this bug does not happen in 1.2.2.
But as other issues surfaced, I hope you backport the fix to 1.0.19 that the modpack (AoE) could update to a new version.

**Comment by sanados** [comment]
Or can you tell me where the satellites in transition (the ones that are started but are still in flight) are stored (.dat file)?
Then npe is thrown when the world is set on the satellite, just need to remove those satellites from the DAT file to get my world working again.

**Comment by zmaster587** [comment]
I might be able to add some compat to ease the transition, what exactly is breaking?  I do know there's a few things that i cannot ease though and may need to be done manually.

Yes, the satellite data is stored in the world/advrocketry/temp.dat  either in each individual world or in dim0 depending on the version, I don't remember which does which anymore.

**Comment by sanados** [comment]
When using the latest version, existing satellites were missing.
Gas Charge Pad was not transferring oxygen to armor (maybe armor issue though)

I checked the temp.dat and found my existing satellites, but not the two that were still in the rocket on the way up.

(reverted to a backup and avoid starting satellites from the space station)

(what might have caused it was that I started the satellite before going down to the planet - so it might think it was in orbit of gallifrey, but the world did not exist yet, was trying to generate the world when starting up the server (modified source and compile an own version) but then I ran into initialization errors.
To auto fix this, it would need a rewrite of the loading/init ... Not worth it, as it is working in 1.2.2.
